src/main/java/teammates/storage/sqlentity/Team.java

```diff
@@ -48,7 +59,7 @@ public Team(Section section, String name) {
         this.setId(UUID.randomUUID());
         this.setSection(section);
         this.setName(name);
-        this.setUsers(new ArrayList<>());
+        this.setTeamToStudentMaps(new ArrayList<>());
```

Comment:
Then these (including lines 109-110, 113-114, and 135 can be changed to use the set of students